src/game.js

```diff
@@ -17,4 +22,49 @@ class Game{
 
         return gameList;
     }
+
+    loadRoles(){
+        let dataFile = fs.readFileSync('./data/roles.json', 'utf8');
+        let roleData = JSON.parse(dataFile);
+        let roleList = [];
+
+        roleData.forEach(element => {
+            roleList.push(new Role(element.roleID, element.title, element.description, element.image, element.color));
+        });
+
+        return roleList;
+    }
+
+    assignPlayerRoles(users){
```

Comment:
Not sure if this should be called `assignRoles()` ,  `createPlayers()`, or `assignPlayerRoles()`

Reply:
Might need to discuss this to see what makes sense